src/pages/blog/magma-release-1.3.1.md

```diff
@@ -0,0 +1,14 @@
+---
+templateKey: blog-post
+title: Magma Release 1.3.1
+author: Phil Ritter
+date: 2020-11-10 T09:00:00.000Z
```

Comment:
The space before T will cause the template rendering to fail unable to parse the timestamp.

Reply:
Fixed timestamp